src/app/(project)/projects/[project]/collections/[collection]/Collection.tsx

```diff
@@ -0,0 +1,182 @@
+                Collections /
+              </Link>
+              <span>{collectionQuery.data?.project?.collection?.metadataJson?.name}</span>
+            </div>
+          </div>
+
+          {/* MAIN */}
+          <div className="mt-8 min-w-full">
+            <div className="flex p-6 bg-stone-900 rounded-lg mt-6">
+              <div className="basis-1/2 min-h-full flex border-r border-r-stone-800 pr-4 gap-4">
+                <img
+                  src={collectionData?.metadataJson?.image as string}
+                  className="w-[80px] h-[80px] aspect-square rounded-lg object-cover"
+                />
+                <div className="flex flex-col gap-2">
+                  <Typography.Header size={Size.H2}>
+                    {collectionData?.metadataJson?.name as string}
+                  </Typography.Header>
+                  <span className="text-sm text-gray-400">
+                    {collectionData?.metadataJson?.description}
+                  </span>
+                </div>
+              </div>
+
+              <div className="basis-1/2 h-full flex flex-col px-4 gap-2 text-sm">
+                <div className="flex items-center justify-between gap-2">
+                  <span className="text-gray-400">Blockchain</span>
+                  <PopoverBox
+                    triggerButton={blockchainIcon}
+                    elements={[<span key="blockchain">{collectionData?.blockchain}</span>]}
+                  />
+                </div>
+                <div className="flex items-center justify-between gap-2">
+                  <span className="text-gray-400">Type</span>
+                  <span>
+                    {collectionData?.drop ? (
+                      <span>
+                        Drop{' '}
+                        <Link
+                          href={`/projects/${project}/drops/${collectionData.drop.id}/mints`}
+                          className="text-yellow-300"
+                        >
+                          view
+                        </Link>
+                      </span>
+                    ) : (
+                      'Open'
+                    )}
+                  </span>
+                </div>
+                <div className="flex items-center justify-between gap-2">
+                  <span className="text-gray-400">Symbol</span>
+                  <span>{collectionData?.metadataJson?.symbol}</span>
+                </div>
+                <div className="flex items-center justify-between gap-2">
+                  <span className="text-gray-400">Created</span>
+                  <span>
+                    {format(convertLocalTime(collectionData?.createdAt), DateFormat.DATE_2)}
+                  </span>
+                </div>
+                <div className="flex items-center justify-between gap-2">
+                  <span className="text-gray-400">Authority</span>
+                  <span>{shorten(collectionData?.createdById)}</span>
```

Comment:
This should be the update authority or the authority for polygon. We aren't saving that in hub-nfts right now so we should drop displaying this for now.

I'll notify @damianr